.github/workflows/mediasoup-rust.yaml

```diff
@@ -49,8 +49,12 @@ jobs:
       - name: cargo clippy
         run: cargo clippy --all-targets -- -D warnings
 
+      # NOTE: In Windows this will build and test libmediasoupworker in release
+      # mode twice since build.rs doesn't allow debug mode for Windows.
       - name: cargo test
-        run: cargo test --verbose
+        run: |
+          cargo test --verbose
+          cargo test --release --verbose
```

Comment:
What do you mean? `cargo test` is debug build, how is building it twice helps?

Reply:
Do you mean that I wrote this?

Reply:
I have absolutely no idea why that duplicate call was added

Reply:
Ah, it just tests debug and release builds, but internal C++ library is only compiled in release mode. Now I get it.

Reply:
And is it ok or does it make sense? I don't remember anything about this.

Reply:
Yes, all good, sorry for the noise